internal/platform/configuration/src/user_settings.rs

```diff
@@ -29,9 +29,15 @@ impl<'a> UserSettings {
     }
 
     pub fn load_configuration(&self, ctx: &mut Context) -> Result<ConfigurationModel> {
-        let mut file = ctx.block_on_with(self.fs_service.read_file(&self.resource))?;
+        // When the file doesn't exist, the code should not panic
```

Comment:
I think this comment is too obvious. Could you please delete it?

Reply:
Yes! I have deleted it!